src/render/quantum.rs

```diff
@@ -504,7 +538,7 @@ impl AudioRenderQuantum {
     }
 
     #[inline(always)]
-    pub fn set_channels_values_at(&mut self, sample_index: usize, values: &[f32]) {
+    pub(crate) fn set_channels_values_at(&mut self, sample_index: usize, values: &[f32]) {
```

Comment:
There something I'd like to test soon with the `AudioBufferSourceNode` that could eventually improve performance and allow to drop this one altogether